GTMSessionFetcher.podspec

```diff
@@ -0,0 +1,51 @@
+# This file specifies the Pod setup for GTMSessionFetcher. It enables developers 
+# to import GTMSessionFetcher via the CocoaPods dependency Manager.
+Pod::Spec.new do |s|
+  s.name        = 'GTMSessionFetcher'
+  s.version     = '1.1.0'
+  s.authors     = 'Google Inc.'
+  s.license     = { :type => 'Apache', :file => 'LICENSE' }
+  s.homepage    = 'https://github.com/google/gtm-session-fetcher'
+  s.source      = { :git => 'https://github.com/google/gtm-session-fetcher.git',
+                    :tag => "v#{s.version}" }
+  s.summary     = 'Google Toolbox for Mac - Session Fetcher.'
+  s.description = <<-DESC
+
+  GTMSessionFetcher makes it easy for Cocoa applications 
+  to perform http operations. The fetcher is implemented 
+  as a wrapper on NSURLSession, so its behavior is asynchronous 
+  and uses operating-system settings on iOS and Mac OS X.
+  DESC
+  
+  s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.8'
+
+  s.compiler_flags = '-DGTM_USE_SESSION_FETCHER=1'
+
+  s.default_subspec = 'Core', 'Full'
```

Comment:
Should this be 'default_subspecs'? It looks like the syntax pluralizes it when there is more than one.